data/Molcas/basicOpenMolcas18.0/MoOCl4_sp.in

```diff
@@ -0,0 +1,17 @@
+&gateway
+ coord
+ 6
+ 
+ Mo     0.000325    -0.000325     0.051810
+ O     0.000206    -0.000206     1.741504
+Cl     1.711727    -1.711727    -0.659846
+Cl    -1.710992    -1.711738    -0.660079
+Cl    -1.711004     1.711004    -0.660311
+Cl     1.711738     1.710992    -0.660079
+ basis = ANO-RCC-MB
```

Comment:
This isn't going to be sufficient to do an ECP calculation. There is no ECP directly associated with ANO basis sets. You should see if Molcas includes the LANL2DZ basis set and ECP, which is what we use for all other `MoOCL4_sp` calculations.

Reply:
It doesn't include a `LANL2DZ` basis set but has different basis sets inside the `basis_library/ecp`. This PR is completely useless, but I tried calculations by usimg basis sets from the `ecp` file but it is giving an error. I have seen a calculation for HAt molecule and it is able to give the needed data so we need to do a similar calculation for MoOCl4. I have mailed those files, please have a look.